spec/lib/jobs/node-discovered-alert-job-spec.js

```diff
@@ -47,21 +49,35 @@ describe("Job.Alert.Node.Discovered", function () {
         it("should _run() pass without additional data", function() {
             this.sandbox.stub(waterline.nodes, 'needByIdentifier').resolves({ type: 'compute' });
             this.sandbox.stub(eventsProtocol, 'publishNodeEvent').resolves();
+	    this.sandbox.stub(lookupService, 'findIpMacAddresses').resolves([
```

Comment:
indent problem. I think you need to check the editor setting, sees your PRs usually have a lot indent problems.